examples/experimental/dagster-airlift/dagster_airlift/mwaa/auth.py

```diff
@@ -32,17 +30,22 @@ def get_session_info(region: str, env_name: str, profile_name: Optional[str]) ->
 
 
 class MwaaSessionAuthBackend(AirflowAuthBackend):
-    def __init__(self, region: str, env_name: str, profile_name: Optional[str] = None):
-        self.region = region
+    def __init__(self, mwaa_client: Any, env_name: str) -> None:
+        self.mwaa_client = mwaa_client
         self.env_name = env_name
-        self.profile_name = profile_name
         # Session info is generated when we either try to retrieve a session or retrieve the web server url
         self._session_info: Optional[Tuple[str, str]] = None
 
+    @staticmethod
+    def for_local(region: str, env_name: str, profile_name: Optional[str] = None):
```

Comment:
nit: maybe call `from_profile` since this reads from a profile in the credentials file which is more broadly useful than local dev since it supports things like inheriting EC2 roles

Reply:
yup make sense